rules/record-teams-meetings/rule.md

```diff
@@ -0,0 +1,40 @@
+---
+type: rule
+title: Do you record Teams meetings?
+uri: record-teams-meetings
+authors:
+  - title: Brady Stroud
+    url: https://ssw.com.au/people/brady-stroud
+  - title: Brady Stroud
```

Comment:
Is one of these authors supposed to be me? :-)

Reply:
yes - fixed 🙂